--- lcms-1.19/jpegicc/jpegicc.c.orig	2009-10-30 16:57:45.000000000 +0100
+++ lcms-1.19/jpegicc/jpegicc.c	2025-10-04 21:41:43.036751125 +0200
@@ -707,7 +707,7 @@ int TransformImage(char *cDefInpProf, ch
        DWORD wInput, wOutput;
        int OutputColorSpace;
        DWORD dwFlags = 0; 
-       DWORD EmbedLen;
+       unsigned int EmbedLen;
        LPBYTE EmbedBuffer;
 
 	   // Observer adaptation state (only meaningful on absolute colorimetric intent)
